pilot/pkg/serviceregistry/aggregate/controller_test.go

```diff
@@ -403,6 +437,40 @@ func TestGetIstioServiceAccounts(t *testing.T) {
 	}
 }
 
+// Test the aggregate controller with federation providers
+func TestFederationGetIstioServiceAccounts(t *testing.T) {
+	federationCtl := buildMockControllerForFederation()
```

Comment:
nit: can we call this `aggregateCtl`? I was confused at first, thinking this was returning a federation controller